htmresearch/frameworks/nlp/classify_htm.py

```diff
@@ -50,6 +52,9 @@ def __init__(self,
     self.retina = retina
     self.apiKey = apiKey
     self.maxSparsity = maxSparsity
+    self.cacheRoot = cacheRoot
```

Comment:
Why store cacheRoot here? It's stored in the encoder - best to get it from there rather than duplicate it. It also doesn't seem to be used anywhere other than initModel - you can just pass cacheRoot in to that method as a parameter.